src/world/ZoneManager.h

```diff
@@ -0,0 +1,35 @@
+#ifndef NONAME_ZONE_H
+#define NONAME_ZONE_H
+
+
+#include <SFML/System.hpp>
+#include <vector>
+
+struct Zone {
+    int x;
+    int y;
+    int w;
+    int h;
+    bool enabled;
+};
+
+class ZoneManager {
+public:
+    const Zone& getZone(int id);
+    void setZone(int id, Zone zone);
+    int addZone(Zone zone);
+
+    bool isInZone(int id, int x, int y);
```

Comment:
Maybe make `Zone` a class and create this method there?

Example:
```cpp
ZoneManager::getZone(2).contains({2.0, 1.0}) // true
```
Also if `Zone` is just a simple struct, an all functionality is implemented in `ZoneManager` then why return it via `getZone()`?

Reply:
I mean, to reduce number of arguments (3 is not that good)

Reply:
BTW, since we have a component system working now and there is a `Position` component, then should we start making methods expect just any object that implements `Position` component instead of just plain `int`s?

Reply:
- Zone as a class won't be so easy to serialize. That's why it is a struct.
- 3 arguments is not much, also if you want, there is overloading using sf::Vector.
- I don't see any usefulness in using PositionComponent as an argument.